src/components/MyAccount/Settings/AccountSettingsDisplayOptions.tsx

```diff
@@ -29,7 +30,50 @@ export const AccountSettingsDisplay = ({ patron }: { patron: Patron }) => {
   return <>{terms.map(buildListElementsWithIcons)}</>
 }
 
-export const AccountSettingsForm = ({ patron }: { patron: Patron }) => {
+export const AccountSettingsForm = ({
+  patron,
+  onValidateForm,
+}: {
+  patron: Patron
+  onValidateForm: (any) => void
+}) => {
+  const [formData, setFormData] = useState({
+    primaryPhone: patron.phones[0]?.number,
+    email: patron.emails[0],
+  })
+
+  const validateForm = useCallback(() => {
```

Comment:
Hmmm I'm on the fence about this pattern. My first instinct was that this logic can be in the `useEffect` below, but I also feel like it's tidy to have it in a method. Something feels redundant in the fact that both hooks have `formData` as a dependency and then the `useEffect` has `validateForm` as a dependency.
@EdwinGuzman what do you think?

whether this ends up as a method or something else, can you change the name to something like `isFormValid`?